web/client/utils/LocaleUtils.js

```diff
@@ -51,6 +51,27 @@ const DATE_FORMATS = {
     "it-IT": "dd/MM/yyyy",
     "nl-NL": "dd/MM/yyyy"
 };
+
+const errorParser = {
+    geostore: {
```

Comment:
I would register this handler in the GeoStoreAPI code, and put here a function to register the parsers